RiotSwiftUI/Modules/Room/TimelinePoll/Coordinator/TimelinePollCoordinator.swift

```diff
@@ -137,7 +139,15 @@ final class PollTimelineCoordinator: Coordinator, Presentable, PollAggregatorDel
                             answerOptions: answerOptions,
                             closed: poll.isClosed,
                             totalAnswerCount: poll.totalAnswerCount,
-                            type: (poll.kind == .disclosed ? .disclosed : .undisclosed),
-                            maxAllowedSelections: poll.maxAllowedSelections)
+                            type: pollKindToTimelinePollType(poll.kind),
+                            maxAllowedSelections: poll.maxAllowedSelections,
+                            hasBeenEdited: poll.hasBeenEdited)
+    }
+    
+    private func pollKindToTimelinePollType(_ kind: PollKind) -> TimelinePollType {
```

Comment:
Could this be a property/method on `PollKind`? Would probably be nicer to `switch` (or use `if`s I can't find `PollKind` now) rather than create an array that's thrown away so quickly.

Reply:
Unfortunately not as PollKind is part of the SDK and TimelinePollType in the SwiftUI target.